plio/io/io_controlnetwork.py

```diff
@@ -279,13 +281,13 @@ def _set_pid(pointid):
                 measure_spec = point_spec.Measure()
                 # For all of the attributes, set if they are an dict accessible attr of the obj.
                 for attr, attrtype in self.measure_attrs:
-                    if attr in g.columns:
-                        setattr(measure_spec, attr, attrtype(m[attr]))
-                measure_spec.serialnumber = m.serialnumber
+                    # Un-mangle common attribute names between points and measures
+                    df_attr = self.measure_field_map.get(attr, attr)
+                    if df_attr in g.columns:
```

Comment:
point_field_map and measure_field_map have 'type' : 'pointType' and 'type' : 'measureType'. So this logic seems to push  type into both measures and points, even though they are set up on different standards. I am going to alter autocnet to simply pass 'pointType' and 'measureType'. This just seems like a sneaky error waiting to happen

Reply:
So this isn't really a comment on this logic, but more about the usefulness of measure_field_map and point_field_map. Since the key values are repeated in both the libraries, I wonder if we shouldn't just be picky about what comes into plio

Reply:
These don't have to do with what comes into PLIO, but rather name collisions in the proto specs. Both the point message and measure message have a type field which will collide when we merge them together.